.github/ISSUE_TEMPLATE/2-bug-report.md

```diff
@@ -0,0 +1,21 @@
+---
+name: "🐛 Bug report"
+labels: bug
+about: Create a bug report to help us improve
+---
+
+**Summary**. Describe the expected and actual behavior here.
+
+## Steps to reproduce
+
+Please clarify how to reproduce the issue.
```

Comment:
You may want to cover two additional points mentioned in #412:

- Mention the code block format in Markdown
A number of users don't know how to paste their XML content correctly.

- Ask whether WinSW is used with Jenkins
This helps us to triage Jenkins module related issues.

Reply:
Agreed